src/game/g_main.pas

```diff
@@ -687,6 +688,9 @@ procedure Init();
     Mix_SetSoundFonts(PChar(e_SoundFont));
   {$ENDIF}
 
+    // Android is not really Unix. Don't try environment variables there, because it may lead to unexpected behaviour and crashes.
+    // On newer Androids, getenv causes game to crash for some reason.
```

Comment:
Please put a line break to maintain the 100 character limit for line length. (1)